site/templates/application/application.jet

```diff
@@ -108,11 +108,11 @@
     {{block navigation()}}
       {{yield nav() }}
     {{end}}
```

Comment:
added this to play around with fixed width but no longer neccesary. will cleanup.